### DISCUSSION TITLE
Can we update kubelet config during bootstrap in AKS

### DISCUSSION THREAD

**Discussion by pavandamarapati** [Azure/AKS#2922] [Q&A] [answered]
In EKS (AWS) there is option to update kubelet config during bootstrap as below, 
https://github.com/awslabs/amazon-eks-ami/blob/master/files/bootstrap.sh 

Is this possible in Azure Kubernetes service?

**Comment by philwelz** [comment, marked as answer]
This is possible. Check the docs for this [here](https://learn.microsoft.com/en-us/azure/aks/custom-node-configuration?tabs=linux-node-pools#kubelet-configuration),
